packages/sheets-formula/src/controllers/formula-editor-show.controller.ts

```diff
@@ -300,7 +281,7 @@ export class FormulaEditorShowController extends Disposable implements IRenderMo
         return false;
     }
 
-    private _refreshArrayFormulaRangeShapeByRanges(unitId: string, subUnitId: string, ranges?: IRange[]): void {
+    private _refreshArrayFormulaRangeShapeByRanges(unitId: string, subUnitId: string): void {
```

Comment:
Since this function do not receive `range` parameter anymore, its name should be changed as well.